default.nix

```diff
@@ -1,10 +1,18 @@
-{ buildGoModule, config, lib, pkgs, installShellFiles, version ? "latest" }:
+{ buildGoModule, config, lib, pkgs, installShellFiles, date, commit }:
 
-buildGoModule {
+buildGoModule rec {
   pname = "process-compose";
-  version = version;
+  version = "0.29.1";
+  pkg = "github.com/f1bonacc1/process-compose/src/config";
```

Comment:
This does not work as a the `buildGoModule` has a limited attribute set (= hash-maps/key-value store) that it accepts. A let binding is more fortunate, see my follow-up PR.